packages/languages/de/README.md

```diff
@@ -19,7 +19,7 @@ import { ZxcvbnFactory } from '@zxcvbn-ts/core'
 import * as zxcvbnCommonPackage from '@zxcvbn-ts/language-common'
 import * as zxcvbnDePackage from '@zxcvbn-ts/language-de'
 
-const password = 'somePassword'
+const password = 'nogleAdgangskode'
```

Comment:
Wrong language pack

Reply:
Woops my bad. Also fixed in the latest commit!